Eatery/Views/BRB/BRBLoginTableViewCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BRBLoginTableViewCell.swift
+//  Eatery
+//
+//  Created by Noah Pikielny on 8/23/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import NVActivityIndicatorView
+import UIKit
+
+class BRBLoginTableViewCell: UITableViewCell {
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+    }
+
+    func configure(stackView: UIStackView) {
+        stackView.removeFromSuperview()
+        stackView.snp.removeConstraints()
+        contentView.addSubview(stackView)
```

Comment:
```suggestion
        stackView.removeFromSuperview()
        contentView.addSubview(stackView)
```
Removing from superview also removes constraints? Maybe different with snapkit